scipy/stats/tests/test_continuous.py

```diff
@@ -1446,6 +1446,38 @@ def test_repr_str_docs(self):
             assert repr(dist(beta=2)) == "HalfGeneralizedNormal(beta=np.float64(2.0))"
         assert 'HalfGeneralizedNormal' in dist.__doc__
 
+    def test_custom_distribution_repr_str(self):
+            # Create a custom distribution class
+            class MyLogUniform:
+                @property
+                def __make_distribution_version__(self):
+                    return "1.16.0"
+
+                @property
+                def parameters(self):
+                    return {'a': {'endpoints': (0, np.inf), 
+                                  'inclusive': (False, False)},
+                            'b': {'endpoints': ('a', np.inf), 
+                                  'inclusive': (False, False)}}
+
+                @property
+                def support(self):
+                    return 'a', 'b'
+
+                def pdf(self, x, a, b):
+                    return 1 / (x * (np.log(b) - np.log(a)))
```

Comment:
For the purposes of this test, it looks like you can just purge out `pdf()` and it will still pass (see below). I suppose we may want to keep the test as simple as possible, even if defining `pdf()` is more realistic in many cases.

```diff
--- a/scipy/stats/tests/test_continuous.py
+++ b/scipy/stats/tests/test_continuous.py
@@ -1464,9 +1464,6 @@ class TestMakeDistribution:
                 def support(self):
                     return 'a', 'b'
 
-                def pdf(self, x, a, b):
-                    return 1 / (x * (np.log(b) - np.log(a)))
-

```